code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -138,10 +138,28 @@
 
 /obj/item/reagent_containers/hypospray/medipen/examine()
 	. = ..()
-	if(reagents?.reagent_list.len)
-		. += span_notice("It is currently loaded.")
+	if(length(reagents?.reagent_list))
+		var/reagent_text = ""
+
+		if(length(reagents.reagent_list) == 1)
+			var/datum/reagent/first = reagents.reagent_list[1]
+			reagent_text = first.name
+
+		else
+			var/len = reagents.reagent_list.len
```

Comment:
```suggestion
			var/len = length(reagents.reagent_list)
```
you should also name this better

Reply:
now that I'm looking at it I don't think that this line is actually used, I left it in there accidentaly I think